Source/WebKit/WebProcess/WebCoreSupport/WebUserMediaClient.cpp

```diff
@@ -46,37 +46,37 @@ void WebUserMediaClient::pageDestroyed()
 
 void WebUserMediaClient::requestUserMediaAccess(UserMediaRequest& request)
 {
-    m_page.userMediaPermissionRequestManager().startUserMediaRequest(request);
+    m_page->userMediaPermissionRequestManager().startUserMediaRequest(request);
 }
 
 void WebUserMediaClient::cancelUserMediaAccessRequest(UserMediaRequest& request)
 {
-    m_page.userMediaPermissionRequestManager().cancelUserMediaRequest(request);
+    m_page->userMediaPermissionRequestManager().cancelUserMediaRequest(request);
 }
 
 void WebUserMediaClient::enumerateMediaDevices(Document& document, UserMediaClient::EnumerateDevicesCallback&& completionHandler)
 {
-    m_page.userMediaPermissionRequestManager().enumerateMediaDevices(document, WTFMove(completionHandler));
+    m_page->userMediaPermissionRequestManager().enumerateMediaDevices(document, WTFMove(completionHandler));
 }
 
 WebUserMediaClient::DeviceChangeObserverToken WebUserMediaClient::addDeviceChangeObserver(WTF::Function<void()>&& observer)
 {
-    return m_page.userMediaPermissionRequestManager().addDeviceChangeObserver(WTFMove(observer));
+    return m_page->userMediaPermissionRequestManager().addDeviceChangeObserver(WTFMove(observer));
 }
 
 void WebUserMediaClient::removeDeviceChangeObserver(DeviceChangeObserverToken token)
 {
-    m_page.userMediaPermissionRequestManager().removeDeviceChangeObserver(token);
+    m_page->userMediaPermissionRequestManager().removeDeviceChangeObserver(token);
 }
 
 void WebUserMediaClient::updateCaptureState(const WebCore::Document& document, bool isActive, WebCore::MediaProducerMediaCaptureKind kind, CompletionHandler<void(std::optional<WebCore::Exception>&&)>&& completionHandler)
 {
-    m_page.userMediaPermissionRequestManager().updateCaptureState(document, isActive, kind, WTFMove(completionHandler));
+    m_page->userMediaPermissionRequestManager().updateCaptureState(document, isActive, kind, WTFMove(completionHandler));
 }
 
 void WebUserMediaClient::setShouldListenToVoiceActivity(bool shouldListen)
 {
-    m_page.send(Messages::WebPageProxy::SetShouldListenToVoiceActivity { shouldListen });
+    m_page->send(Messages::WebPageProxy::SetShouldListenToVoiceActivity { shouldListen });
```

Comment:
Why don't we need a `protectedPage()` to call these functions on the page? here and above? Still doesn't seem completely safe.

Reply:
Oh yeah, we most certainly need to use `protectedPage` for these. Not sure why I missed it.